portfolio/src/main/webapp/script.js

```diff
@@ -14,13 +14,32 @@
 
 async function getData() {
     const response = await fetch('/data');
-    const comments = await response.json();
+    const json = await response.json();
+    const comments = json.comments;
     let html = "";
     for(const comment of comments) {
         html += "<div class=\"comment\">";
```

Comment:
In a case like this, I find people frequently mix `'`, `"`, and `` ` `` as their string character. I don't know if that's a good practice (there are a lot of tools that complain if you mix `'` with `"`, but in this case it's definitely appealing to do `html += '<div class="comment">'` so you can avoid all of the quote escaping.

Reply:
Yeah I should've just changed all of the concatenations at once when you showed me how to use `\``. I changed them now.